jest.config.js

```diff
@@ -1,11 +1,10 @@
 module.exports = {
   clearMocks: true,
-  moduleFileExtensions: ['js', 'ts'],
-  testEnvironment: 'node',
-  testMatch: ['**/*.test.ts'],
-  testRunner: 'jest-circus/runner',
```

Comment:
jest-circurs has been the default test runner since jest 27
https://jestjs.io/blog/2021/05/25/jest-27